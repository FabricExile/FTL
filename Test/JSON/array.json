[]
[43,7.98E+3,-5.12e-27,4.12381823858512,"\"]",{"fred":[5,6]}]
