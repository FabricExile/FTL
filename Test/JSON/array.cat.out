[]
[
  43,
  7980.0,
  -5.12e-27,
  4.12381823858512,
  "\"]",
  {
    "fred" : [
      5,
      6
      ]
    }
  ]
